speeduino/comms.ino

```diff
@@ -1641,9 +1641,10 @@ void receiveCalibration(byte tableID)
   bool every2nd = true;
   int x;
   int counter = 0;
-  pinMode(LED_BUILTIN, OUTPUT); //pinMode(13, OUTPUT);
+  // stm32 board has buildin led used as fuel pump
```

Comment:
Can this be removed from the code? I see no real benefit other than debugging. When sending the Thermistor calibration the STM32 just froze. It took me forever to find out it had todo with analogwrite(LED_BUILDIN ..) or digitalWrite(LED_BUILTIN, LOW).

Reply:
Yes, I had included that 2 years ago for debugging STM32 without st-link as the code frozen on some parts of code. With st-link debugging is pretty easy though

Reply:
Do you recommend any manuals to get debugging with st-link v2 working under platformIO? If i set a breakpoint i get the error the program stopped for debugging!

Reply:
I'm using VScode for a while, it is much faster than PIO, but if you like it VScode have a PIO plugin too. On VScode I have to include some lines manually and you need openocd v0.10 , older versions don't work

Reply:
I got debugging working in the end. It was simple, change the default build to the black_stm32 environment. I use VScode also but with the PlatformIO plugin to build arduino code. To much work to set it up oterwise :-D.